shell.nix

```diff
@@ -60,6 +58,10 @@ mkShellMold {
     pkgs.jq # used in ./tests and ./tools
     pkgs.nixfmt-rfc-style # to format this file
     pkgs.npins # used in tools/nix/update.py
+    pkgs.iconv
```

Comment:
Could you please leave a short comment on why `iconv` is needed / which dependency wants it?

Reply:
Done